tests/unit/test_charm.py

```diff
@@ -119,11 +119,6 @@ def test_check_slurmdbd_slurm_not_active(self, *_) -> None:
         self.harness.charm._check_slurmdbd(max_attemps=1)
         self.assertEqual(self.harness.charm.unit.status, BlockedStatus("Cannot start slurmdbd"))
 
-    def test_on_db_unavailable(self) -> None:
```

Comment:
This was removed since `on_db_available` was removed but later we should add tests for the replacements being `database_created` and `endpoints_changed`.

Reply:
So I decided to remove `endpoints_changed` since I was able to get the router working after all. As for adding new tests, theoretically we are covered by the integration tests (integration tests will fail if the handler is not working), but we can add more unit tests at a later day. I plan on reworking the unit tests after we decouple from slurm_ops_manager as I think this will help reduce the amount of function side-effects.